lib/kitchen/directions/bake_toc.rb

```diff
@@ -60,14 +60,15 @@ def self.li_for_composite_chapter(chapter)
         HTML
       end
 
-      def self.li_for_chapter(chapter)
+      def self.li_for_chapter(chapter, cases:)
         pages = chapter.element_children.only(PageElement, CompositePageElement)
         inner_composite_chapters = chapter.element_children.only(CompositeChapterElement)
 
         <<~HTML
           <li class="os-toc-chapter" cnx-archive-shortid="" cnx-archive-uri="">
             <a href="##{chapter.title.id}">
-              <span class="os-number"><span class="os-part-text">#{I18n.t(:chapter)} </span>#{chapter.count_in(:book)}</span>
+              <span class="os-number"><span class="os-part-text">#{I18n.t("chapter#{'.nominative' \
+              if cases}")} </span>#{chapter.count_in(:book)}</span>
               <span class="os-divider"> </span>
               <span class="os-text" data-type="" itemprop="">#{chapter.title.first!('.os-text').text}</span>
             </a>
```

Comment:
Why do we need target labels in TOC? I'm guessing that there might be links to toc in content, because we are coping module/chapter titles with their ids. But target label should link us directly to module, not to TOC. I'm not sure how we can fix that problem :thinking:

Reply:
Ok nevermind :smile: Their not target labels, these are changes needed after modyfications in `pl.yml` file. Sorry for the confusion :sweat_smile: